config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
A path like `/var/lib/clearcontainers/runtime/bundles/pause_bundle/rootfs/bin/pause` would probably be more inline with the FHS (although I don't like that it hard-codes the bundle name).

Going forward, I can't see why -- as you say -- we can't just read the bundle path from the config file. The pause bin path would then be relative to that. The current relative path (`/bin/pause`) should probably be changed to make it "more unique" as it's currently a pretty generic name. Something like `/usr/libexec/clearcontainers/runtime/pause-container` for example would be more descriptive and much less likely to conflict with existing bundle contents.

Finally, shouldn't there be a check on whether the bundle directory part of `defaultPauseBinPath` exists, since if it doesn't it's not (ever?) going to be valid.